collect.py

```diff
@@ -241,19 +237,29 @@ def run_tests(firefox_driver, chrome_driver):
     chrome_driver.quit()
 
 
+no_of_threads = 5
```

Comment:
This is OK for the configuration, we don't need a separate file or anything.
Just put it at the top of the file and make it uppercase, `THREADS_NUM = 5`.